index.html

```diff
@@ -1,11 +1,549 @@
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>HTML Homework 6</title>
-</head>
-<body>
-
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WebStudio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:ital,wght@0,400;0,500;0,900;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <a href="https://dumanskyy.github.io/HTML_Homework-4" class="logo">
+            <span class="logo__web">Web</span>
+            <span class="logo__studio">Studio</span>
+          </a>
+          <nav class="header__nav">
+            <ul class="nav__list">
+              <li class="nav__item ">
+                <a class="nav__link is-active" href="./index.html">Студія</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="./portfolio.html">Портфоліо</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#">Контакти</a>
+              </li>
+            </ul>
+          </nav>
+          <div class="contacts">
+            <a href="mailto:info@devstudio.co" class="contacts__email">
+              <svg class="contacts__email-icon">
+                <use href="./images/svg/icons.svg#envelope" />
+              </svg>
+              info@devstudio.com</a
+            >
+            <a href="tel:+380961111111" class="contacts__mobile">
+               <svg class="contacts__mobile-icon">
+                <use href="./images/svg/icons.svg#smartphone" />
+              </svg>
+
+              +38 096 111 11 11
+            </a>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <main>
+      <section class="hero">
+        <div class="container">
+          <div class="hero__content">
+            <h1 class="hero__title">ЕФЕКТИВНІ РІШЕННЯ ДЛЯ ВАШОГО БІЗНЕСУ</h1>
+            <button data-open-modal name="button" class="hero__button">Замовити послугу</button>
+          </div>
+        </div>
+      </section>
+
+      <section class="features">
+        <div class="container">
+          <ul class="features__list">
+            <li class="features__list-item">
+              <!-- [ x] FIXME Change to 'features__list-image' -->
+              <div class="features__list-image">
+                <svg class="list__item-image">
+                  <use href="./images/svg/icons.svg#antenna" />
+                </svg>
+              </div>
+              <h3 class="feature__title">УВАГА ДО ДЕТАЛЕЙ</h3>
```

Comment:
if the styles for each section are identical, create a section element  -  section__title and pass all the styles to it. In such way, you won't have to write identical code more than once (DRY principle)